__tests__/index.ts

```diff
@@ -294,6 +294,25 @@ describe('Mongo', () => {
     });
   });
 
+  describe('findUserByServiceId', () => {
+    it('should return null for not found user', async () => {
+      const ret = await mongo.findUserByServiceId('facebook', 'invalid');
+      expect(ret).not.toBeTruthy();
+    });
+
+    it('should return user', async () => {
+      const userId = await mongo.createUser(user);
+      let ret = await mongo.findUserByServiceId('facebook', '1');
+      expect(ret).not.toBeTruthy();
+      await mongo.setService(userId, 'facebook', { id: '1' });
+      ret = await mongo.findUserByServiceId('facebook', '1');
+      await delay(10);
```

Comment:
I'm curious what is the delay for? I've never used this pattern before when writing these sort of tests.

Reply:
Here it make no sense indeed but these delay where introduced because the tests where failing between two queries the id inserted where not found in the db, that was really weird